editor/editor_properties.cpp

```diff
@@ -1856,13 +1857,19 @@ EditorPropertyTransform::EditorPropertyTransform() {
 ////////////// COLOR PICKER //////////////////////
 
 void EditorPropertyColor::_color_changed(const Color &p_color) {
+	if (!Input::get_singleton()->is_mouse_button_pressed(BUTTON_LEFT))
+		emit_changed(get_edited_property(), p_color, "", true);
+	else {
+		CanvasItem* p = dynamic_cast<CanvasItem*>(get_edited_object());
```

Comment:
`Sprite3D` also has `modulate` property which inherited by `Spaital` not `CanvasItem`.
what about it?

Reply:
Have no idea why but it's working

![image](https://user-images.githubusercontent.com/37383316/67249404-eee9e880-f43d-11e9-9ad2-fd5f9094b93a.png)

Reply:
Oh wait... When you are moving the mouse over the colors it doesn't change anything unless you release LMB

Reply:
and need to check usages under `SpatialMaterial` like Albedo/Color, Emission/Emission, Transmission/Transmission.

Reply:
Fixed